entrypoint.sh

```diff
@@ -5,16 +5,15 @@ set -o errexit    # always exit on error
 set -o pipefail   # honor exit codes when piping
 set -o nounset    # fail on unset variables
 
-git clone https://github.com/electron/dependent-repos
-cd module
-npm install
 npm run build
-npm test
+
 [[ `git status --porcelain` ]] || exit
+
 git add .
-git config user.email "zeke@sikelianos.com"
-git config user.name "Zeke Sikelianos"
-git commit -am "update database"
-# npm version minor -m "bump minor to %s"
+git config user.email "electron@github.com"
+git config user.name "Electron Bot"
+git commit -am "chore: update dependent-repos database"
+
 git push origin master --follow-tags
-# npm publish
\ No newline at end of file
+
+curl $SNITCH_URL
```

Comment:
This should be a secret in the workflow config right?

Reply:
oh yes you right